osf/models/user.py

```diff
@@ -456,10 +457,17 @@ def social_links(self):
         social_user_fields = {}
         for key, val in self.social.items():
             if val and key in self.SOCIAL_FIELDS:
-                if not isinstance(val, basestring):
-                    social_user_fields[key] = val
+                if isinstance(self.SOCIAL_FIELDS[key], basestring):
+                    if isinstance(val, basestring):
+                        social_user_fields[key] = self.SOCIAL_FIELDS[key].format(val)
+                    else:
+                        # Provide a single account url for services which we allow multiple-accounts
```

Comment:
Good place for a comment, but I think small tweaks can make it more clear what you're doing

- "Only return the first URL for services where we allow multiple accounts", or similar.